lmdeploy/lite/apis/auto_awq.py

```diff
@@ -10,36 +10,11 @@
                                             smooth_layers)
 from lmdeploy.lite.utils import collect_target_modules
 
-from .calibrate import calibrate
+from .calibrate import LAYER_TYPE_MAP, NORM_TYPE_MAP, calibrate
 
 # from lmdeploy.lite.utils.export_turbomind import export_turbomind_config
 
-LAYER_TYPE_MAP = {
-    'InternLMForCausalLM': 'InternLMDecoderLayer',
-    'InternLM2ForCausalLM': 'InternLM2DecoderLayer',
-    'QWenLMHeadModel': 'QWenBlock',
-    'Qwen2ForCausalLM': 'Qwen2DecoderLayer',
-    'BaiChuanForCausalLM': 'DecoderLayer',  # Baichuan 7B
-    'BaichuanForCausalLM': 'DecoderLayer',  # Baichuan2 7B
-    'LlamaForCausalLM': 'LlamaDecoderLayer',
-    'LlavaLlamaForCausalLM': 'LlamaDecoderLayer',
-    'MGMLlamaForCausalLM': 'LlamaDecoderLayer',  # mini gemini
-    'InternLMXComposer2ForCausalLM': 'InternLM2DecoderLayer',
-    'ChatGLMForConditionalGeneration': 'GLMBlock',
-}
-NORM_TYPE_MAP = {
-    'InternLMForCausalLM': 'InternLMRMSNorm',
-    'InternLM2ForCausalLM': 'InternLM2RMSNorm',
-    'QWenLMHeadModel': 'RMSNorm',
-    'Qwen2ForCausalLM': 'Qwen2RMSNorm',
-    'BaiChuanForCausalLM': 'RMSNorm',  # Baichuan 7B
-    'BaichuanForCausalLM': 'RMSNorm',  # Baichuan2 7B
-    'LlamaForCausalLM': 'LlamaRMSNorm',
-    'LlavaLlamaForCausalLM': 'LlamaRMSNorm',
-    'MGMLlamaForCausalLM': 'LlamaRMSNorm',  # mini gemini
-    'InternLMXComposer2ForCausalLM': 'InternLM2RMSNorm',
-    'ChatGLMForConditionalGeneration': 'RMSNorm',
-}
+NORM_TYPE_MAP = NORM_TYPE_MAP  # legency
```

Comment:
legency -> legacy 
But I don't understand why we should make the `NORM_TYPE_MAP=NORM_TYPE_MAP` statement

Reply:
NORM_TYPE_MAP is not used in this file, unused import will lead to lint error. And I don't know if other files will reference it or not.

Reply:
Get it. May fix the typo "legency -> legacy"